protocol.js

```diff
@@ -116,20 +117,26 @@ module.exports = function (stateless) {
     return function (err, stream, state) {
       if(err) return cb(err)
 
-      var en_nonce = state.remote.app_mac.slice(0, 24)
-      var de_nonce = state.local.app_mac.slice(0, 24)
+      var encryptNonce = state.remote.app_mac.slice(0, 24)
+      var decryptNonce = state.local.app_mac.slice(0, 24)
 
       cb(null, {
         remote: state.remote.publicKey,
         //on the server, attach any metadata gathered
         //during `authorize` call
         auth: state.auth,
+        crypto: {
```

Comment:
I had to add this to the callback to be able to pass the shs1-test suite.
I've not documented these.
This seems ok?